Podfile.lock

```diff
@@ -1,32 +1,32 @@
 PODS:
-  - AppSyncRealTimeClient (1.9.0):
+  - AppSyncRealTimeClient (1.10.0):
```

Comment:
does this need to be updated to latest 3.0.0?

Reply:
Updated it